packages/fast-components-styles-msft/src/utilities/high-contrast.ts

```diff
@@ -16,18 +16,22 @@ export enum HighContrastColor {
     background = "Background",
 }
 
+function ImportantColor(color: HighContrastColor): string {
```

Comment:
typically capitol letters are reserved for class constructors, enums, interfaces, and type aliases - can you rename to `importantColor`?

You should also consider abstracting - the only assumption this code really makes is that the argument type is a string. You could move this to a generic utility file and change the type signature to simply expect a string value. Then this could be used to add important to any css value

Reply:
abstract the function to its own utility